torch/nn/functional.py

```diff
@@ -1165,17 +1165,16 @@ def embedding(input, weight, padding_idx=None, max_norm=None, norm_type=2,
 
     Args:
         input (LongTensor): Tensor containing indices into the embedding matrix
-        weight (Tensor): The embedding matrix
-            Number of rows should correspond to the maximum possible index + 1,
-            number of columns is the embedding size
+        weight (Tensor): The embedding matrix with number of rows equal to the maximum possible index + 1,
+            and number of columns equal to the embedding size
         padding_idx (int, optional): If given, pads the output with the embedding vector at :attr:`padding_idx`
                                          (initialized to zeros) whenever it encounters the index.
-        max_norm (float, optional): If given, will renormalize the embedding vectors to have a norm lesser than
-                                    this before extracting. Note: this will modify :attr:`weight` in-place.
+        max_norm (float, optional): If given, each embedding vector with norm larger than max_norm
```

Comment:
Actually, can you use `` :attr:`max_norm` `` please? it will be formatted nicer.

Reply:
Yes sure, I hope it is now as you intended.

Reply:
Awesome! Thanks!